samples/crypto/pbkdf2/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,5 @@
+# Enable both oberon driver and hardware crypto accelerator
+# Key derivation is only supported in software driver but is using chained
+# driver to accelerate sub-operations.
```

Comment:
`# driver which will use hardware accelerated crypto for low-level crypto operations, if enabled.`